### PR TITLE
D7: The new website.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Upgrade all the things!
- Write some new documentation
- Update some old documentation
- Add new Examples
- Change URLs and LInks

# Needed By (Date)
- October 31st

# Steps to Test

1. Review [Preview Build Site](https://deploy-preview-143--dev-decanter-v7.netlify.app/)
2. Review Code
3. Make any comments to the PR you feel that are required
4. Call out anything you have questions on or would consider a blocker

# Affected Projects or Products
- Decanter 7
- Decanter 6

# TODO

- [x] Fix up some of the examples
- [x] Review PR for differences (see if I broke/dropped anything)
- [ ] Get all yall feedback
- [x] Awesome lists updates

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16
\ No newline at end of file
+20
```
Comment (sherakama) [inline .nvmrc:1]: Might as well go to 20

File: components/Cta/CtaGroup.styles.ts [hunk 2]
```diff
@@ -3,18 +3,18 @@ import { CtaGroupProps } from './CtaGroup';
...
-  profile: 'su-leading-none children:su-leading-none children:su-pr-10 children:su-mr-10 children:su-border-r children:su-border-black-70 last:children:su-pr-0 last:children:su-mr-0 last:children:su-border-r-0 su-text-18', // Add/Edit & Delete buttons on profile cards
-  form: 'su-flex su-justify-center su-w-full su-rs-mt-7 su-relative su--left-13 md:su--left-15', // Cancel and Save buttons at the bottom of most forms
-  'form-register': 'su-flex su-justify-center su-w-full su-rs-mt-6',
-  modal: 'su-flex su-justify-center su-rs-mt-3 children:su-rs-mr-1 last:children:su-mr-0', // Modal Buttons
-  'image-form': 'su-flex su-flex-col children:su-rs-mb-0 sm:su-flex-row sm:children:su-rs-mr-1 last:children:su-mr-0',
+  card: 'children:mr-16 last:children:mr-0', // Add/Edit & Delete buttons on data cards
+  profile: 'leading-none children:leading-none children:pr-10 children:mr-10 children:border-r children:border-black-70 last:children:pr-0 last:children:mr-0 last:children:border-r-0 text-18', // Add/Edit & Delete buttons on profile cards
+  form: 'flex justify-center w-full rs-mt-7 relative -left-13 md:-left-15', // Cancel and Save buttons at the bottom of most forms
```
Comment (sherakama) [inline components/Cta/CtaGroup.styles.ts:14]: Fix this `-left-13` `-left-15`

File: components/HeroIcon/HeroIcon.styles.ts [hunk 3]
```diff
@@ -85,44 +75,41 @@ export const iconMap = {
...
+  'academic-cap': 'w-08em -mt-02em',
+  'arrow-left': 'w-08em -mt-02em',
+  'arrow-right': 'w-08em -mt-02em',
+  camera: 'w-1em -mt-02em',
+  download: 'w-08em -mt-02em',
+  back: 'w-1em -mt-02em',
+  'chevron-right': 'w-1em -mt-02em',
+  'chevron-down': 'w-1em -mt-01em',
+  email: 'w-[1.2em]',
+  external: 'w-08em -rotate-45 group-hover:-rotate-45 group-focus:-rotate-45',
```
Comment (sherakama) [inline components/HeroIcon/HeroIcon.styles.ts:99]: fix this `-rotate`

File: components/Link/Link.tsx [hunk 4]
```diff
@@ -10,33 +10,15 @@ export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(
...
-      as,
-      replace,
-      scroll,
-      shallow,
-      passHref,
-      prefetch,
-      locale,
       children,
-      // And pass the rest to the anchor
-      ...anchorProps
```
Comment (sherakama) [inline components/Link/Link.tsx:23]: New next link doesn't require the `<a>` nested tag

File: components/SidebarNav/SidebarNavData.ts [hunk 5]
```diff
@@ -25,7 +25,6 @@ export const SidebarNavData = [
       { label: 'Stanford identity and brand compliance', href: routes.stanfordIdentity() },
       { label: 'Accessibility', href: routes.accessibilityNotes() },
       { label: 'Type, grids and color', href: routes.typeGridsAndColor() },
-      { label: 'Patterns and examples', href: routes.patternsAndExamples() },
```
Comment (sherakama) [inline components/SidebarNav/SidebarNavData.ts:28]: This page has no content in it, nor could I find any in our existing resources.

File: content/examples/cards.mdx [hunk 6]
```diff
@@ -3,165 +3,165 @@ title: Cards
...
-              <h3 className="su-font-serif su-font-bold su-leading-display su-tracking-normal su-type-0 su-mt-0 su-relative su-inline su-text-white">Galápagos Islands</h3>
+          <div className="rs-pt-1 rs-px-2 rs-pb-3">
+            <a href="/travel-study/destinations/galapagos-family-2023/" className="stretched-link group z-20 mb-08em type-2 no-underline hocus:underline underline-offset-[3px] decoration-[0.12em] decoration-digital-red-light focus:outline-none text-white">
+              <h3 className="font-serif font-bold leading-display tracking-normal type-0 mt-0 relative inline text-white">Galápagos Islands</h3>
               <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true"
-                   className="su-transition su-w-08em su-ml-03em su--mt-02em su-transform-gpu group-hover:su-translate-x-02em group-focus:su-translate-x-02em su-relative su-inline-block group-hover:su-text-white group-focus:su-text-white">
+                   className="transition w-08em ml-03em -mt-02em transform-gpu group-hover:translate-x-02em group-focus:translate-x-02em relative inline-block group-hover:text-white group-focus:text-white">
```
Comment (sherakama) [inline content/examples/cards.mdx:34]: Fix `-mt-02em`

File: content/examples/cards.mdx [hunk 7]
```diff
@@ -3,165 +3,165 @@ title: Cards
...
+           className="stretched-link group z-20 rs-mt-2 mb-02em no-underline hocus:underline underline-offset-[3px] decoration-[0.12em] decoration-digital-red-light focus:outline-none type-1 text-black hocus:text-black">
+          <h3 className="font-serif font-bold leading-display tracking-normal relative inline type-0">
             <span>Asian American and Pacific Islander Heritage Month at Stanford</span>
           </h3>
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true"
-               className="su-transition su-w-08em su-ml-02em su--rotate-45 group-hover:su--rotate-45 group-focus:su--rotate-45 su-transform-gpu group-hover:su-translate-x-01em group-focus:su-translate-x-01em group-hover:su--translate-y-01em group-focus:su--translate-y-01em su-relative su-inline-block su-text-digital-red-light">
+               className="transition w-08em ml-02em -rotate-45 group-hover:-rotate-45 group-focus:-rotate-45 transform-gpu group-hover:translate-x-01em group-focus:translate-x-01em group-hover:-translate-y-01em group-focus:-translate-y-01em relative inline-block text-digital-red-light">
```
Comment (sherakama) [inline content/examples/cards.mdx:99]: Fix more negative classes

File: content/examples/cards.mdx [hunk 8]
```diff
@@ -3,165 +3,165 @@ title: Cards
...
```
Comment (sherakama) [inline content/examples/cards.mdx:132]: fix negative classes

File: content/examples/cards.mdx [hunk 9]
```diff
@@ -3,165 +3,165 @@ title: Cards
...
-            <a className="su-flex su-w-fit su-group su-transition-colors su-no-underline su-underline-offset-[3px] hocus:su-underline su-text-digital-red-light hocus:su-text-digital-red" href="/career-connections/">
+        <div className="rs-mt-2">
+          <div className="block print:hidden text-left text-19 md:text-21 xl:text-23">
+            <a className="flex w-fit group transition-colors no-underline underline-offset-[3px] hocus:underline text-digital-red-light hocus:text-digital-red" href="/career-connections/">
               <span>See career resources</span>
               <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true"
-                   className="su-transition su-w-08em su-ml-03em su--mt-02em su-transform-gpu group-hover:su-translate-x-02em group-focus:su-translate-x-02em su-inline-block su-text-digital-red-light group-hover:su-text-digital-red group-focus:su-text-digital-red">
+                   className="transition w-08em ml-03em -mt-02em transform-gpu group-hover:translate-x-02em group-focus:translate-x-02em inline-block text-digital-red-light group-hover:text-digital-red group-focus:text-digital-red">
```
Comment (sherakama) [inline content/examples/cards.mdx:164]: fix negative here too

File: content/examples/cards.mdx [hunk 10]
```diff
@@ -172,22 +172,22 @@ lastUpdatedDate: 2023-05-25T12:00:00.000Z
...
```
Comment (sherakama) [inline content/examples/cards.mdx:190]: fix negative here too

File: content/examples/form-elements.mdx
Comment (sherakama) [inline content/examples/form-elements.mdx:1]: These inputs all look the same on the front end.

File: contentlayer.config.ts [hunk 11]
```diff
@@ -4,7 +4,6 @@ import { defineDocumentType, makeSource } from 'contentlayer/source-files';
 import remarkGfm from 'remark-gfm';
 // Rehype packages
 import rehypePrismPlus from 'rehype-prism-plus';
-import rehypePresetMinify from 'rehype-preset-minify';
```
Comment (sherakama) [inline contentlayer.config.ts:7]: Re-establish minify here

File: public/brand-bar-options.png
Comment (sherakama) [inline public/brand-bar-options.png:1]: Not sure why this was removed. Should bring this back.

Comment (sherakama) [inline public/brand-bar-options.png:1]: Not sure why this was removed. Should bring this back.

File: public/universal-footer.png
Comment (sherakama) [inline public/universal-footer.png:1]: Not sure why this was removed. Should bring this back.

**Comment by sherakama** [comment]
This PR is nearing a place where it will be ready for review. Feel free to get your input in early.

Once this is done we will be able to move it to the production Decanter website and move the existing one to a Version 6 domain. After that, I will be moving the version 7 branch to the main branch in the core Decanter project and v6 will become a secondary branch so that the default will be to download version 7.

**Comment by sherakama** [comment]
Hey everyone, I'm putting a review by date on this PR for October 31st. After which, I will begin the switch. Please call out any blockers before then.

**Comment by kerri-augenstein** [comment]
Hey Shea,

Is there a dev link where I can review the site?

Kerri Augenstein
she/her/hers
Senior UX/UI Designer, Lead Creative
Stanford | University IT
505 Broadway St. | Cardinal Hall, 4th Floor | Redwood City, Ca. 94063
***@***.******@***.***> | 401-486-9525
________________________________

**Comment by sherakama** [comment]
@kerri-augenstein here you go: https://deploy-preview-143--dev-decanter-v7.netlify.app/

**Comment by yvonnetangsu** [comment]
@sherakama I know I said I'll try to get to this today, but I think it's going to be next Monday - sorry about that! The only thing I can think of is perhaps a slight tweak to the examples for modular typography so it aligns more with what we usually use it for. The site looks awesome - thank you for all the work (and the new examples) 🎉 😄 !

**Comment by sherakama** [comment]
> The site looks awesome

That's all @jdwjdwjdw and @rebeccahongsf for their work. I've just been filling out some content.

**Comment by sherakama** [comment]
@kerri-augenstein 

> On the "website" where we show elements like [colors](https://decanter-v7.netlify.app/#color-palette), [typography](https://decanter-v7.netlify.app/#text), etc., would it make sense to provide a "back to Decanter" link up in the top left?

I guess we could, but really, those links were put there to have _something_ in that section. I went back and looked at our content doc and pulled what I could from that. https://docs.google.com/document/d/1-HNdgVy0ME1zlXdbmr6O9wzGPDEmoxdta8fUHS_0r-c/edit

Please have another review.

> For the padding, on the inside of the content of our pages, could we go to padding-left: 10rem (instead of 8rem)

This has been fixed.

> I'll have a content edit to the "Figma" page, but need to solicit support for that from David, as our edu accounts are no longer free. Adobe, weeeeeee.

Good luck.

**Comment by sherakama** [comment]
Alrighty then. Today is the day. Boldly going forward with the understanding that changes can still happen.

**Pull request merged**